src/main/java/frc/robot/commands/leftIntakeGoTo.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.Constants;
+
+public class leftIntakeGoTo extends CommandBase {
+  /** Creates a new IntakeGoTo. */
+  public double endpoint;
+ //tracking initial start point so we can compare during climbing to see when to finish
+  public double leftStartpoint;
+  public double rightStartpoint;
+  public leftIntakeGoTo(double Endpoint) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    endpoint = Endpoint;
+
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    RobotContainer.m_intakeSubsystem.stopActuators();
+    leftStartpoint = RobotContainer.m_intakeSubsystem.currentLeftActuatorPosition();
```

Comment:
You should NOT be trying to control a subsystem by calling back thru RobotContainer!

The subsystems you need should be passed into the Command as inputs, saved in private members and then referenced via private members of the Command.

This also applies to all methods in this file.